erpnext/setup/doctype/company/company.json

```
@@ -736,13 +738,29 @@
    "fieldtype": "Link",
    "label": "Default Use Tax Payable Account",
    "options": "Account"
+  },
+  {
+   "depends_on": "eval:!doc.__islocal",
```

```suggestion
   "depends_on": "eval:!doc.__islocal",
```
any reason to hide it when new doc

---

Not really, I had copied the configuration from the prior row in the site. I'll fix this.